src/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -1596,6 +1597,16 @@ public Territory getCurrentBattleTerritory() {
   public IBattle getCurrentBattle() {
     return m_currentBattle;
   }
+
+  protected void raids() {
+    // Remove air raids - this adds all the bombing raids to the battle tracker
+    for( final Territory t : m_battleTracker.getPendingSBRSites() ) {
```

Comment:
The user point of view is a bit irrelevant. There is actually an anti-pattern called "feature-envy" that you have here. It is about data encapsulation, and namely that battle tracker does not have it. It's an example of procedural code where an external class pulls the inner data of another class, then processes it and passes it back.

WRT method location. Please read chapter 5 of clean code, please see the standard. This method is 1500 lines away from where it is used. I'm not even sure the method itself is a useful delimiter, I would just as well have it inlined, instead it's buried in this giant class file. The location is seemingly random. 

And yes, do make it private, minimum scope of variables and methods is expected.